app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,48 @@
+class RestaurantsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index
+    Restaurant.sorted
```

Comment:
Probably best being explicit about how this sorts, even in the controller level. One way would be to call the scope `sorted_by_date`, another can be to just call `order(created_at: :desc)` at this level. IMO you should try not to force presentation related logic on your models, this tends to create a lot of mildly different scopes over time and confusion

Reply:
Noted. I will do that in the future. For now, I only changed the name of the scope (and kept the scope just for practice).